### PR TITLE
Added new animation - words dazing

### PR DESCRIPTION
**Pull request by siddoinghisjob**

### CODE REVIEW

**Comment by siddoinghisjob** [comment]
Can you please accept this pr?

File: animations/WordComingDazzing.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.heading{
```
Comment (NiallEccles) [inline animations/WordComingDazzing.css:1]: Update this to be your animation name `.WordComingDazzing`

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for making this PR.

Can you address the changes I've requested?

**Comment by siddoinghisjob** [comment]
Have made the change 😸

**Comment by NiallEccles** [comment]
Nice work 👏

**Pull request merged**